MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,15 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.96.0 ➞ v0.97.0
+
+### *(new feature)* snowflake_stream_on_table resource
+
+To enhance clarity and functionality, the new resource `snowflake_stream_on_table` has been introduced to replace the previous `snowflake_stream`. Recognizing that the old resource carried multiple responsibilities within a single entity, we opted to divide it into more specialized resources.
```

Comment:
are there any differences between old resource and new resource/resources (besides of course being dedicated to the given object)? If so, it would be nice to guide our user what they should change if they already have `snowflake_stream` and would like to make `snowflake_stream_on_table`  out of it (e.g. import syntax changes, right?).

Reply:
It's really straightforward to migrate to the new resource—just change the resource to stream_on_table and move the table ID properly—so I've decided not to include this in the migration guide.

Reply:
But even such a sentence is helpful if someone does not know what are the changes